cmd/commands/subvolume.go

```diff
@@ -0,0 +1,41 @@
+package command
+
+import (
+	"github.com/rook/kubectl-rook-ceph/pkg/subvolume"
+	"github.com/spf13/cobra"
+)
+
+// SubvolumeCmd represents the rook command to manage subvolumes
+var SubvolumeCmd = &cobra.Command{
+	Use:   "stale-subvolume",
+	Short: "manage subvolumes",
+	Args:  cobra.ExactArgs(1),
+}
+
+var listCmd = &cobra.Command{
+	Use:                "ls",
+	Short:              "Print the list of stale subvolumes no longer in use.",
+	DisableFlagParsing: true,
+	Args:               cobra.MaximumNArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		clientsets := GetClientsets(cmd.Context())
+		VerifyOperatorPodIsRunning(cmd.Context(), clientsets, OperatorNamespace, CephClusterNamespace)
+		subvolume.List(cmd.Context(), clientsets, CephClusterNamespace, OperatorNamespace)
+	},
+}
+
+var deleteCmd = &cobra.Command{
+	Use:                "delete",
+	Short:              "Deletes the stale subvolume.",
+	DisableFlagParsing: true,
+	Args:               cobra.MaximumNArgs(2),
+	Run: func(cmd *cobra.Command, args []string) {
+		clientsets := GetClientsets(cmd.Context())
+		VerifyOperatorPodIsRunning(cmd.Context(), clientsets, OperatorNamespace, CephClusterNamespace)
+	},
```

Comment:
```suggestion
	Run: func(cmd *cobra.Command, args []string) {
		clientsets := GetClientsets(cmd.Context())
		VerifyOperatorPodIsRunning(cmd.Context(), clientsets, OperatorNamespace, CephClusterNamespace)
	},
```

implementation for delete is pending?

Reply:
yes, will send in next commit, also one more check is pending here.